.eslintrc.js

```diff
@@ -32,6 +32,10 @@ const ELASTIC_LICENSE_HEADER = `
  */
 `;
 
+const corePublicPattern = './src/core/public/(?!utils|index).*';
```

Comment:
As of v1 of `contains-path`+regex is no longer supported, do you think it would make sense to use `multimatch` instead?

```js
const coreServerPatterns = ['src/core/server/**/*', '!src/core/server/index.js']
// ...
from: [
  ...coreServerPatterns
]
```

Reply:
I think we can. although I didn't want to diverge implementation from original one.